Code/PgSQL/rdkit/CMakeLists.txt

```diff
@@ -0,0 +1,156 @@
+set(EXTENSION "rdkit")
```

Comment:
I can also check/fix OSX on Friday.  We have already made a similar approach for avalon we will have to merge with this model anyway.